graylog2-web-interface/src/views/components/AdaptableQueryTabs.tsx

```diff
@@ -219,42 +229,55 @@ const AdaptableQueryTabs = ({ maxWidth, queries, titles, selectedQueryId, onRemo
           {tabTitle}
         </NavItem>
       ));
+
+      queriesList.push({
+        id,
+        title,
+      });
```

Comment:
I didn't do that because:

-  in this place we already iterating queries. 
- `AdaptableQueryTabsConfiguration` don't need all information about queries, just `id` and `title`
- Also, here we are generating titles for unnamed pages, if we would do that one more time in AdaptableQueryTabsConfiguration we would have some kind of 2 sources of true